README.md

```diff
@@ -16,93 +17,23 @@ The **VMware.vSphereDSC** module contains resources for:
 - **vCenter** and **Host** settings
 
 For more information about all available **Resources**, please visit the **wiki** page of the repository: [Wiki](https://github.com/vmware/dscr-for-vmware/wiki).
+For more information about the **VMware.vSphereDSC** module please read here: [VMware.vSphereDSC info](https://github.com/vmware/dscr-for-vmware/blob/master/vSphereDsc.md).
 
-## Getting Started
+The **VMware.PSDesiredStateConfiguration** module provides an alternative in-language way to compile and execute DSC Configurations. It does not require the use of LCM and supports powershell core.
```

Comment:
provides commands to compile and execute DSC Configuration without using the DSC Local Configuration Manager. Compiled configuration is stored in memory as PS Object. Set, Get, Test-... cmdlets use Invoke-DSCResource cmdlet of PSDesiredStateConfiguration module to execute/run/process the compiled configuration. 

Remove powershell core